checkov/terraform/plan_parser.py

```diff
@@ -238,13 +240,24 @@ def _get_resource_changes(template: dict[str, Any]) -> dict[str, dict[str, Any]]
     """Returns a resource address to resource changes dict"""
 
     resource_changes_map = {}
-
     resource_changes = template.get("resource_changes")
+
     if resource_changes and isinstance(resource_changes, list):
-        resource_changes_map = {
-            change.get("address", ""): change
-            for change in resource_changes
-        }
+        for each in resource_changes:
```

Comment:
can you rename `each` to something like `resource` or `changed_resource`?
Will be more readable in python code (even though I guess it comes from terraform conventions)